packages/components/src/interfaces/complex/Hero.ts

```diff
@@ -64,7 +64,10 @@ export const HeroSchema = Type.Composite(
     HeroBackgroundImageSchema,
     Type.Object({
       type: Type.Literal("hero", { default: "hero" }),
-      variant: Type.Literal("gradient", { default: "gradient" }),
+      variant: Type.Union([Type.Literal("gradient"), Type.Literal("block")], {
+        default: "gradient",
+        title: "Hero banner style",
+      }),
```

Comment:
nit: need to specify the title of the option otherwise it will show up as "anyOf-0".

```suggestion
      variant: Type.Union(
        [
          Type.Literal("gradient", { title: "Gradient" }),
          Type.Literal("block", { title: "Block" }),
        ],
        {
          default: "gradient",
          title: "Hero banner style",
        },
      ),
```

Can see something [similar done for InfoCards](https://github.com/opengovsg/isomer/blob/9c5a9f6dd811e0828781df59563654463ae8fa2f/packages/components/src/interfaces/complex/InfoCards.ts#L40-L48).

Reply:
done in https://github.com/opengovsg/isomer/pull/717/commits/554417c549c2ebbddbaac8c4e3de4e54a8209c38! thank you